### DISCUSSION TITLE
We should add videos to slideshow...

### DISCUSSION THREAD

**Discussion by frederikhors** [Valexr/Slidy#29] [Ideas]
We should add videos to slideshow...

**Comment by Valexr** [comment]
in Slidy you have default slot for slide content – this is [simple example](https://ru.svelte.dev/repl/1ac00a7d69c4409ebb499e614d61133c) of integration video

sizes, tags for video type and playback logic at your discretion

**Comment by frederikhors** [comment]
I mean something like https://vegas.jaysalvat.com.

I would like to have a slideshow with images and videos.

**Comment by Valexr** [comment]
I think you don’t need Slidy carousel logic for this case. Yours example slideshow can be realise by pure css animations.

**Comment by frederikhors** [comment]
Do you know what can I use?

**Comment by Valexr** [comment]
https://tympanus.net/Tutorials/CSS3FullscreenSlideshow/

**Comment by frederikhors** [comment]
What a pity that there is nothing like Vegas for Svelte, comfortable, without losing your head with hand-held CSS.

**Comment by Valexr** [comment]
good idea... I'm will think about implementing different animation type...

**Comment by frederikhors** [comment]
Yes please. Vegas is for Jquery, I would like to use Svelte with same amazing effect.

**Comment by Valexr** [comment]
Effects In Vegas writing on pure css 🤓

**Comment by frederikhors** [comment]
Yeah, I know. But using Vegas take me 2 minutes to create animations. Using css I loose my mind. Are you interested in create something like Vegas for Svelte? With all effetcs alredy pre-built in Svelte? With timings and all?

**Comment by Valexr** [comment]
Not now, but maybe later & what?
